.github/workflows/test.yml

```
@@ -0,0 +1,149 @@
+
+on:
+  pull_request:
+    branches: [ "master", "main" ]
+    paths:
+      - 'core/**'
+      - 'storage/**'
+      - 'support/**'
+      - 'build.gradle'
+      - 'settings.gradle'
+  push:
+    branches: [ "master" ]
```

YAMLlint (1.38.0)

[error] too many spaces inside brackets

(brackets)